<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_private_element_dialog</class>
 <widget class="QDialog" name="add_private_element_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="group_box_element_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>325</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Value</string>
   </property>
   <widget class="QTextEdit" name="text_edit_element_value">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>33</y>
      <width>256</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_box_attribute">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>325</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Attribute</string>
   </property>
   <widget class="QLineEdit" name="line_edit_group_hex">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>33</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_attribute_group">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>64</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_attribute_name">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_attribute_name">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>90</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_attribute_element">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>64</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Byte</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_element_hex">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>33</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>152</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_private_block">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>291</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>Private</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Creator</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_creator_list">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>add_private_element_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>add_private_element_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
